pkg/drivers/response.go

```diff
@@ -0,0 +1,100 @@
+package drivers
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/MontFerret/ferret/pkg/runtime/core"
+	"github.com/MontFerret/ferret/pkg/runtime/values"
+	"github.com/MontFerret/ferret/pkg/runtime/values/types"
+)
+
+// HTTPResponse HTTP response object.
+type HTTPResponse struct {
+	StatusCode int
+	Status     string
+	Headers    HTTPHeaders
+}
+
+func (resp *HTTPResponse) Type() core.Type {
+	return HTTPResponseType
+}
+
+func (resp *HTTPResponse) String() string {
+	return resp.Status
+}
+
+func (resp *HTTPResponse) Compare(other core.Value) int64 {
+	if other.Type() != HTTPResponseType {
+		return Compare(HTTPResponseType, other.Type())
+	}
+
+	// it's a safe cast because HTTPResponse doesn't implement
```

Comment:
does NOT?

Reply:
But *HTTPResponse implements

Reply:
add more descriptive comment https://github.com/MontFerret/ferret/pull/391/commits/338bcc5cd43e2554c1189263cb910abd9db68f8e